pages/in-vitro-systems/[id].js

```diff
@@ -135,6 +143,12 @@ export async function getServerSideProps({ params, req }) {
           filterErrors: true,
         })
       : [];
+    const pooledFrom = inVitroSystem.pooled_from
```

Comment:
Optionally, I would also check the length of `pooled_from` is non-zero so that you don’t request zero objects. Not sure a zero-length `pooled_from` can happen though.